Packs/Whois/Integrations/Whois/Whois.py

```diff
@@ -8273,6 +8273,7 @@ def create_outputs(whois_result, domain, query=None):
     ec = {'Name': domain}
     standard_ec = {}  # type:dict
     standard_ec['WHOIS'] = {}
+    standard_ec['QueryResult'] = 'NOT FOUND' not in str(whois_result.get('raw', 'NOT FOUND'))
```

Comment:
are you sure that with this implementation this goes under `Domain.Whois` and not `Domain` ?

Reply:
Right